plugins/net_plugin/net_plugin.cpp

```diff
@@ -2413,8 +2417,19 @@ namespace eosio {
                fc_elog( logger, "Error accepting connection: ${m}", ("m", ec.message()));
                // For the listed error codes below, recall start_listen_loop()
                switch (ec.value()) {
+                  case EMFILE: // same as boost::system::errc::too_many_files_open
+                  {
+                     // no file descriptors available to accept the connection. Wait on async_timer
+                     // and retry listening using shorter 100ms timer than SHiP or http_plugin
+                     // as net_pluging is more critical
+                     accept_error_timer->expires_from_now(boost::posix_time::milliseconds(100));
+                     accept_error_timer->async_wait([self = shared_from_this()]( const boost::system::error_code &ec) {
```

Comment:
Not seeing the need for the shared_ptr capture here

Reply:
right. updated, thanks!